apps/protocol-frontend/src/components/DeleteContributionDialog.tsx

```diff
@@ -0,0 +1,91 @@
+import { useRef, useEffect } from 'react';
+import {FiTrash2 } from 'react-icons/fi';
+import {UIUser} from '@govrn/ui-types';
+import {
+    AlertDialog,
+    AlertDialogBody,
+    AlertDialogFooter,
+    AlertDialogContent,
+    AlertDialogOverlay,
+    IconButton,
+    Button,
+    useDisclosure,
+  } from '@chakra-ui/react';
+
+  
+interface DeleteContributionDialogProps {
+  row: string;
+  userData: UIUser;
+  deleteContribution: (string) => void;
+  DialogSize : string;
+}
+
+const  DeleteContributionDialog = ({
+    row,
+    userData,
+    deleteContribution,
+    DialogSize,
+    }:DeleteContributionDialogProps) => {
+
+    const { isOpen, onOpen,  onClose } = useDisclosure()
+    const cancelRef = useRef<HTMLButtonElement>(null);
+    
+
+  const handleDeleteContribution = async (contribution_id: number | string) => { 
+    const y = await deleteContribution(contribution_id);
+    
+  };
+  
+    return (
+      <>
+     
+        <IconButton
+            icon={<FiTrash2 fontSize="1rem" />}
+            variant="ghost"
+            color="gray.800"
+            disabled={
+             row.original.user.id !== userData?.id ||
+             row.original.status === 'minted'
+           }
+           aria-label="Delete Contribution"
+           onClick={onOpen}
+        
+        />
+        <AlertDialog
+          isOpen={isOpen}
+          leastDestructiveRef={cancelRef}
+          onClose={onClose}
+          size = {DialogSize}
+        >
+          <AlertDialogOverlay >
+            <AlertDialogContent bgColor="brand.primary.50" color="gray.800">
```

Comment:
lets use `bgColor='white'` like our other modals in the UI

Reply:
Addressed bgColor